docs/CHANGELOG.asciidoc

```diff
@@ -10,18 +10,124 @@
 
 Review important information about the {kib} 8.0.0 releases.
 
+* <<release-notes-8.0.0>>
 * <<release-notes-8.0.0-rc2>>
 * <<release-notes-8.0.0-rc1>>
 * <<release-notes-8.0.0-beta1>>
 * <<release-notes-8.0.0-alpha2>>
 * <<release-notes-8.0.0-alpha1>>
 
 --
+[[release-notes-8.0.0]]
+== {kib} 8.0.0
+
+coming::[8.0.0]
+
+Review the {kib} 8.0.0 changes, then use the <<upgrade-assistant,Upgrade Assistant>> to complete the upgrade.
```

Comment:
8.0 doesn't have an Upgrade Assistant.  Best to point to the doc in the 7.17 branch.  

```suggestion
Review the {kib} 8.0.0 changes, then use the {kibana-ref-all}/7.17/upgrade-assistant.html[Upgrade Assistant] to complete the upgrade.
```

Reply:
This line appears several times in this doc.